onnxruntime/core/session/inference_session.h

```diff
@@ -191,7 +192,10 @@ class InferenceSession {
     * Calling this API is optional.
     * @return OK if success.
     */
-  common::Status FilterEnabledOptimizers(const std::unordered_set<std::string>& optimizers_to_disable)
+  common::Status FilterEnabledOptimizers(const InlinedHashSet<std::string>& optimizers_to_disable)
+      ORT_MUST_USE_RESULT;
```

Comment:
>ORT_MUST_USE_RESULT [](http://example.com/codeflow?start=6&length=19)

don't need this for things returning Status
would it be easier to just have one function taking InlinedHashSet by value?

Reply:
The problem with functions taking such big things by value is that everyone would forget to move when it is appropriate and we get an extra copy.